source/_data/data.json

```diff
@@ -291,5 +291,20 @@
     "long": "20",
     "short": "20"
   },
-  "seconds": "31"
+  "seconds": "31",
+  "attributes": {
+    "Attribute()": {
+      "class": []
+    }
+  },
```

Comment:
Decided to add this globally, which will simplify templates.  This way we don't have to always check and use `create_attribute()` for the top level attributes.